server/src/command_table_create.go

```diff
@@ -357,6 +364,7 @@ func commandTableCreate(s *Session, d *CommandData) {
 	}
 
 	t := NewTable(d.Name, s.UserID())
+	t.PreGameVisible = preGameVisible
```

Comment:
we don't need to add a new property here, do we?
just use the existing t.Visible, and then set it to true after the start starts in the command_table_restart function

Reply:
oh yeah that should work, will do. but the function should still keep the preGameVisible argument right?

Reply:
yes because you need to emulate whats happening on command_replay_create.go line 84